docs/src/overview/build-deploy.md

```diff
@@ -1,118 +1,80 @@
+Hold requests
+: The system holds incoming requests so there is no service interruption.
 
-Because container configuration (for both your application and its underlying services) is exclusively based on your configuration files
-and your configuration files are managed through Git,
-we know that a given container has a 1:1 relationship with a Git commit.
-That means builds are always repeatable.
-It also means that if we detect that there are no changes that would affect a given container,
-we can skip the build step entirely and reuse the existing container image, saving a great deal of time.
+Unmount current containers
+: The containers gets disconnected from their file system mounts.
 
-In practice, the entire build process usually takes less than a minute.
+Mount file systems
+: The containers that were disconnected in the previous step, connect the file system to the new containers.
 
-## Deploying the application
+If it's a new branch and doesn't have an existing file system, it gets cloned from the parent branch.
+Note that these shouldn't be directories with code since it would be a security risk.
 
-Deploying the application also has several steps, although they're much quicker.
-
-First, we pause all incoming requests and hold them so that there's no interruption of service.
-Then we disconnect the current containers from their file system mounts, if any,
-and connect the file systems to the new containers instead.
-If it's a new branch and there is no existing file system, we clone it from the parent branch.
-
-We then open networking connections between the various containers,
-but only those that were specified in the configuration files (using the `relationships` key).
-No configuration, no connection.
-That helps with security, as only those connections that are actually needed even exist.
+Expose services
+: The system opens the networking connections between the various containers that were specified in the configuration files (using the `relationships` key).
+That helps with security, as only the specified connections exist.
 The connection information for each service is available in an application as environment variables.
 
-Now that we have working, running containers there are two more steps to run.
-First, if there is a `start` command specified in your `.platform.app.yaml` file to start the application, we run that.
-(With PHP, this is optional.)
-
-Then we run your deploy hook.
-Just like the build hook, the deploy hook is any number of shell commands you need to prepare your application.
-Usually this includes clearing caches, running database migrations, and so on.
-You have complete access to all services here as your application is up and running,
-but the file system where your code lives is now read-only.
+Run start commands
+: It’s technically optional for all runtimes. The only requirement is that you must have `web.commands.start` or `web.locations` and `hooks.deploy`.
```

Comment:
This doesn't seem to describe what's being done. It seems more fitting for a how-to article. Here, I would describe what the purpose of these commands are rather than how to do them.